src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java

```diff
@@ -108,6 +111,14 @@
             common=true)
     protected ValidationStringency VALIDATION_STRINGENCY = ReadConstants.DEFAULT_READ_VALIDATION_STRINGENCY;
 
+    @Argument(
+            doc = "Minimum raw number of reads that must be present to even be considered as a het.",
```

Comment:
maybe "Minimum raw number of reads that must be present at a site for it to be considered for het compatibility." instead?